woocommerce/woocommerce-android
# Shipping lines design recommendations

Closes: #11637

### Description
This PR fixes the design recommendations from the Shipping lines project

> I think we should show the banner above the total section – but we should auto-close the total section when the “Add shipping” sheet is shown. This way we avoid what’s happening in your second image. Showing it on top of the Collect Payment button would block access to it, and potentially cause accidental taps if we make the banner auto-hide.

> Either way, I think adding an icon next to the link calls attention a little more.

![image](https://github.com/woocommerce/woocommerce-android/assets/18119390/0322bbc8-e80b-47d1-aba7-fa815496e3be)

### Testing information
1. Open the app
2. Navigate to orders
3. Tap on Add new order (+)
4. Add a product to enable the shipping section
5. Tapp on Add shipping
6. Add a shipping line
7. Check that the feedback banner is shown on top of the Orders Total section
8. Check that the feedback banner CTA contains the material review icon
9. Check that expanding the total section collapses the feedback banner

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/9aa57e04-3407-4c8f-9910-70ba495c3dfe
